storybook/webpack.config.ts

```diff
@@ -140,10 +129,17 @@ const removeSvgFromTest = rule => {
   }
 }
 
-module.exports = ({ config }) => {
+const excludeExternal = (rule: Rule): Rule => ({
+  exclude: /node_modules\/(?!(\@cultureamp)).*/,
+  ...rule,
+})
+
+export default ({ config }) => {
   // Storybook's base config applies file-loader to svgs
   config.module.rules = config.module.rules.map(removeSvgFromTest)
-  config.module.rules.push(...rules)
-  config.resolve.extensions.push(...extensions)
+  config.module.rules.push(
+    [babelRule, styles, svgs, svgIcons, elm].map(excludeExternal)
+  )
+  config.resolve.extensions.push([".ts", ".tsx"])
```

Comment:
I _think_ you'll want to spread this. `config.resolve.extensions.push(...[".ts", ".tsx"])`

Reply:
Yes! Thank you! 🙏